packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.spec.js

```diff
@@ -184,6 +186,22 @@ describe('Component > InteractionLayer', function () {
         wrapper.simulate('pointermove', fakeEvent)
         expect(fakeEvent.target.setPointerCapture.withArgs('fakePointer')).to.have.been.calledOnce()
       })
+
+      describe('onPointerDown when creating', function () {
+        it ('should call the handlePointerDown function', function () {
+          const fakeEvent = {
+            pointerId: 'fakePointer',
+            type: 'pointer',
+            target: {
+              setPointerCapture: sinon.stub(),
+              releasePointerCapture: sinon.stub()
+            }
+          }
+          wrapper.find(StyledRect).simulate('pointerdown', fakeEvent)
+          wrapper.find(StyledRect).simulate('pointerdown', fakeEvent)
+          expect(activeTool.handlePointerDown).to.have.been.calledOnce()
+        })
```

Comment:
What this spec is testing is slightly confusing. I think this is testing that `handlePointerDown` is called only once, which is why `pointerdown` is simulated twice? Perhaps the description should be `should call the handlePointerDown function only once`?